discovery-provider/src/queries/notifications.py

```diff
@@ -726,6 +731,66 @@ def notifications():
             track_id = entry.track_id
             owner_info[const.tracks][track_id] = owner
 
+        # Get playlist updates
+        today = date.today()
+        thirty_days_ago = today - timedelta(days=30)
+        thirty_days_ago_time = datetime(
+            thirty_days_ago.year,
+            thirty_days_ago.month,
+            thirty_days_ago.day,
+            0,
+            0,
+            0
+        )
+        playlist_update_query = session.query(Playlist)
+        playlist_update_query = playlist_update_query.filter(
+            Playlist.is_current == True,
+            Playlist.is_delete == False,
+            Playlist.blocknumber > min_block_number,
+            Playlist.blocknumber <= max_block_number)
+
+        playlist_update_results = playlist_update_query.all()
+
+        # Represents all playlist update notifications
+        playlist_update_notifications = []
+        playlist_update_notifs_by_playlist_id = {}
+        for entry in playlist_update_results:
+            if entry.last_added_to < thirty_days_ago_time:
+                continue
+            playlist_update_notifs_by_playlist_id[entry.playlist_id] = {
+                const.notification_type:
+                const.notification_type_playlist_update,
+                const.notification_blocknumber: entry.blocknumber,
+                const.notification_timestamp: entry.created_at,
+                const.notification_initiator: entry.playlist_owner_id,
+                const.notification_metadata: {
+                    const.notification_entity_id: entry.playlist_id,
+                    const.notification_entity_type: 'playlist',
+                    const.notification_playlist_update_timestamp: entry.last_added_to
+                }
+            }
+
+        users_that_favorited_playlists_dict = ft.reduce(
```

Comment:
maybe comment here that this is a dict of
playlist_id => user_id[]
as well as comment on that we need this to determine which users should be notified?